model/account.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Account struct {
+	ID       uint   `gorm:"primary_key"`
+	User     string `gorm:"not null;unique"`
+	Password string `gorm:"not null"`
```

Comment:
We should use `bcrypt`, `mcrypt` or any other decent password handling implementation. This will also ensure a level of security when handling brute force attacks. Those libraries automatically embeds the salt into the password value.